cli/spec/kontena/cli/stacks/validate_command_spec.rb

```diff
@@ -5,26 +5,71 @@
   include ClientHelpers
   include RequirementsHelper
   include FixturesHelpers
+  include OutputHelpers
 
   context 'without dependencies' do
     before do
       [:read, :exist?].each do |meth|
         allow(File).to receive(meth).with(fixture_path('kontena_v3.yml')).and_call_original
         allow(File).to receive(meth).with(fixture_path('docker-compose_v2.yml')).and_call_original
+        allow(File).to receive(meth).with(fixture_path('stack-with-liquid.yml')).and_call_original
       end
     end
 
-    it 'validates a yaml file' do
-      expect{Kontena.run!('stack', 'validate', fixture_path('kontena_v3.yml'))}.not_to exit_with_error
-      expect{Kontena.run!('stack', 'validate', fixture_path('kontena_v3.yml'))}.to output(/stack:.*version:.*services:.*variables:/m).to_stdout
+    it 'outputs interpolated YAML' do
+      expect{Kontena.run!('stack', 'validate', fixture_path('stack-with-liquid.yml'))}.to output_yaml(
+        'stack' => 'user/stackname',
+        'version' => '0.1.1',
+        'variables' => {
+          'grid_name' => hash_including(
+              'value' => 'validate stackname',
```

Comment:
Very odd, because the `{{ GRID }}` -> `validate` shows up correctly here. But not in the actual variables, or the interpolated envs